client/my-sites/site-settings/main.jsx

```diff
@@ -1,127 +1,202 @@
+import GeneralSettings from './section-general';
+import WritingSettings from './form-writing';
+import DiscussionSettings from './section-discussion';
+import AnalyticsSettings from './section-analytics';
+import ImportSettings from './section-import';
+import ExportSettings from './section-export';
+import SiteSecurity from './section-security';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import i18n from 'lib/mixins/i18n';
+
+/**
+ * Module vars
+ */
+const debug = debugFactory( 'calypso:my-sites:site-settings' );
+
+export class SiteSettingsComponent extends Component {
+	constructor( props ) {
+		super( props );
+
+		// bound methods
+		this.updateSite = this.updateSite.bind( this );
+
+		this.state = {
+			site: this.props.sites.getSelectedSite()
+		};
+	}
+
+	componentWillMount() {
 		debug( 'Mounting SiteSettings React component.' );
-		this.props.sites.on( 'change', this._updateSite );
-	},
+		this.props.sites.on( 'change', this.updateSite );
+	}
 
-	componentWillUnmount: function() {
-		this.props.sites.off( 'change', this._updateSite );
-	},
+	componentWillUnmount() {
+		this.props.sites.off( 'change', this.updateSite );
+	}
 
-	getImportPath: function() {
-		var site = this.state.site,
-			path = '/settings/import';
+	getImportPath() {
+		var { site } = this.state;
+		let path = '/settings/import';
 
 		if ( site.jetpack ) {
-			return site.options.admin_url + 'import.php';
+			return `${ site.options.admin_url }import.php`;
 		}
 
 		return [ path, site.slug ].join( '/' );
-	},
+	}
 
-	getExportPath: function() {
-		var site = this.state.site;
-		return site.jetpack ? site.options.admin_url + 'export.php' : '/settings/export/' + site.slug;
-	},
+	getExportPath() {
+		var { site } = this.state;
+		return site.jetpack
+			? `${ site.options.admin_url }export.php`
+			: `/settings/export/${ site.slug }`;
+	}
 
-	getStrings: function() {
+	getStrings() {
 		return {
-			general: this.translate( 'General', { context: 'settings screen' } ),
-			writing: this.translate( 'Writing', { context: 'settings screen' } ),
-			discussion: this.translate( 'Discussion', { context: 'settings screen' } ),
-			analytics: this.translate( 'Analytics', { context: 'settings screen' } ),
-			security: this.translate( 'Security', { context: 'settings screen' } ),
-			'import': this.translate( 'Import', { context: 'settings screen' } ),
-			'export': this.translate( 'Export', { context: 'settings screen' } ),
+			general: i18n.translate( 'General', { context: 'settings screen' } ),
+			writing: i18n.translate( 'Writing', { context: 'settings screen' } ),
+			discussion: i18n.translate( 'Discussion', { context: 'settings screen' } ),
+			analytics: i18n.translate( 'Analytics', { context: 'settings screen' } ),
+			security: i18n.translate( 'Security', { context: 'settings screen' } ),
+			import: i18n.translate( 'Import', { context: 'settings screen' } ),
+			export: i18n.translate( 'Export', { context: 'settings screen' } ),
```

Comment:
I disagree with this. Since at lesat ES5, [using reserved words as property names is safe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Reserved_word_usage). Babel should take care of quoting those prop names if we're worried about older browsers that don't support this behavior (which would probably only be like <= IE9, which we don't care about anyways).

From that same page @rralian linked to:

> Reserved words actually only apply to Identifiers (vs. IdentifierNames). As described in es5.github.com/#A.1, these are all IdentifierNames which do not exclude ReservedWords.
>
> ``` js
> a.import
> a["import"]
> a = { import: "test" };
> ```